gallery/src/styles/NxThreatNumber/NxThreatNumberPage.tsx

```diff
@@ -8,38 +8,38 @@ import React from 'react';
 
 import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
 
-const nxThreatNumberTableExampleCode = require('!!raw-loader!./NxThreatNumberTableExample.html').default,
-    nxThreatNumberListExampleCode = require('!!raw-loader!./NxThreatNumberListExample.html').default;
+import NxThreatNumberBasicExample from './NxThreatNumberBasicExample';
+import NxThreatNumberListExample from './NxThreatNumberListExample';
+
+const nxThreatNumberBasicExampleCode = require('!!raw-loader!./NxThreatNumberBasicExample').default,
+    nxThreatNumberListExampleCode = require('!!raw-loader!./NxThreatNumberListExample').default;
 
 const NxThreatNumberPage = () =>
   <>
     <GalleryDescriptionTile>
       <p className="nx-p">
-        When an IQ Policy Threat Number is displayed adjacent to an <code className="nx-code">NxThreatBar</code>,
-        style the number with <code className="nx-code">nx-threat-number</code>
-      </p>
-      <p className="nx-p">
-        Note that if a threat bar is used inside <code className="nx-code">nx-cell</code> the {''}
-        <code className="nx-code">nx-cell--threat-bar</code> modifier must be applied to the table cell. If that cell
-        might have multi-line content, that content would need to be wrapped in a custom class to restore the padding
-        removed by <code className="nx-code">nx-cell--threat-bar</code>.
+        When an IQ Policy Threat Number is displayed adjacent to an <code className="nx-code">NxThreatIndicator</code>,
+        style the number with <code className="nx-code">.nx-threat-number</code>
       </p>
     </GalleryDescriptionTile>
 
+    <GalleryExampleTile title="Basic Example"
+                        id="nx-threat-number-basic-example"
+                        liveExample={NxThreatNumberBasicExample}
+                        codeExamples={nxThreatNumberBasicExampleCode}>
+      An example of <code className="nx-code">.nx-threat-number</code> used
+      adjacent to an <code className="nx-code">NxThreatIndicator</code> in a simple inline layout. This how how the
+      two would be used together within a table cell, for instance.
+    </GalleryExampleTile>
+
     <GalleryExampleTile title="List Example"
```

Comment:
I'm questioning whether we still even need `.nx-threat-number`.  In this example, it actually doesn't do anything since the list text already has the same styles.  In tables, there's an `nx-cell--label` class that also can be used to apply those same styles.  So I'm wondering if we should get rid of `.nx-threat-number` and just rely on those classes in tables and lists. @glenhunter ?

Reply:
> I'm questioning whether we still even need `.nx-threat-number`. In this example, it actually doesn't do anything since the list text already has the same styles. In tables, there's an `nx-cell--label` class that also can be used to apply those same styles. So I'm wondering if we should get rid of `.nx-threat-number` and just rely on those classes in tables and lists. @glenhunter ?

I think we should probably keep the class, but I don't see any reason that it shouldn't be folded into this example as an option.